packages/nc-gui/components/virtual-cell/components/ListItem.vue

```diff
@@ -31,10 +33,12 @@ interface Attachment {
   mimetype: string
 }
 
-const attachments: Attachment[] = computed(() => {
+const attachments: ComputedRef<Attachment[]> = computed(() => {
   try {
-    if (attachment && row[attachment.title]) {
-      return typeof row[attachment.title] === 'string' ? JSON.parse(row[attachment.title]) : row[attachment.title]
+    if (props.attachment && props.attachment[props.attachment.title]) {
```

Comment:
the logic is different. why change `row` to `props.attachment`?

Reply:
typo